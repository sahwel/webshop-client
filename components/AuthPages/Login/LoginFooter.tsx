import Link from 'next/link';
import Button from '../../common/Form/Button/Button';

const LoginFooter = () => {
  return (
    <div>
      <Button children="Sign in" type="submit" />
      <Link href="/register" passHref>
        <a className="text-center mt-2 cursor-pointer block">
          Donâ€™t have an account? <span className="font-bold">Register</span>
        </a>
      </Link>
    </div>
  );
};

export default LoginFooter;
