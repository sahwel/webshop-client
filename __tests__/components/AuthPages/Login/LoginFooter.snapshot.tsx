import { render } from '@testing-library/react';
import LoginFooter from '@/components/AuthPages/Login/LoginFooter';

describe('LoginFooter snapshot', () => {
  it('renders LoginFooter unchanged', () => {
    const { container } = render(<LoginFooter />);
    expect(container).toMatchSnapshot();
  });
});
