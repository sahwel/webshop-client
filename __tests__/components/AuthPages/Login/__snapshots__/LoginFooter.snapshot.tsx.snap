// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginFooter snapshot renders LoginFooter unchanged 1`] = `
<div>
  <div>
    <button
      class="button"
      type="submit"
    >
      Sign in
    </button>
    <a
      class="text-center mt-2 cursor-pointer block"
      href="/register"
    >
      Donâ€™t have an account? 
      <span
        class="font-bold"
      >
        Register
      </span>
    </a>
  </div>
</div>
`;
